cli/command/image/history_test.go

```diff
@@ -0,0 +1,101 @@
+package image
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"regexp"
+	"testing"
+	"time"
+
+	"github.com/docker/docker/api/types/image"
+	"github.com/docker/docker/cli/internal/test"
+	"github.com/docker/docker/pkg/testutil/assert"
```

Comment:
We just merged #32614 which removes this package. It was replaced with https://godoc.org/github.com/stretchr/testify/assert. So you should be able to change these asserts to the new package.

For "assert error contains" there is `testutil.ErrorContains()`